README.md

```diff
@@ -1,10 +1,20 @@
 ## What is Dagger?
 
-Dagger is a CI/CD engine that is **programmable** and **container-native**.
+Dagger is a programmable CI/CD engine that runs your pipelines in containers.
 
-* **Programmable**: develop your CI/CD pipelines as code, in the same programming language as your application.
+### Programmable
 
-* **Container-native**: Dagger executes your pipelines entirely as [standard OCI containers](https://opencontainers.org), so the same pipeline can run on your local machine, a CI runner, a dedicated server, or any container hosting service. Other benefits of container-native pipelines include superior caching; cross-language instrumentation; compatibility with the massive Docker ecosystem; and more.
+Develop your CI/CD pipelines as code, in the same programming language as your application.
+
+### Runs your pipelines in containers
+
+Dagger executes your pipelines entirely as [standard OCI containers](https://opencontainers.org/). This has several benefits:
+
+* **Instant local testing**
```

Comment:
Why not explain this point?

Reply:
To leave you something to do :) Seriously, I couldn't think of a good wording right away, but we should add it soon.

Reply:
Alright!